docs/platform/what-is-tizen/versions/tizen-5-5-m2.md

```diff
@@ -0,0 +1,49 @@
+ML API sets are based on NNStreamer (https://github.com/nnsuite/nnstreamer), which allow to construct GStreamer pipelines consisting of conventional GStreamer media plugins and neural network plugins. Although we plan to expand its functionality to on-device training in later releases, ML API sets of the current version of Tizen support machine learning inferences only. In ML API sets, there are two API subsets: Pipeline ML API set and Single ML API set.
+Pipeline ML API set allows developers to construct and execute GStreamer pipelines with the conventional GStreamer plugins and neural network plugins provided by NNStreamer. With this API set, developers may construct pipelines with multiple instances of neural network models, input streams, and various stream path manipulators. Note that a few of GStreamer plugins that are white-listed are allowed with Tizen Application APIs; on the other hand, all of NNStreamer plugins are allowed. For input streams, you may use camera, mic, files, or arbitrary inputs from applications. However, your applications need to acquire appropriate privileges for the given inputs. For output streams, you may use files or callback functions of the application.
+Single ML API set allows developers to invoke a given neural network model (e.g., .tflite TensorFlow-lite model file) with a single instance of input buffer easily.
+In Tizen 5.5, ML API sets allow TensorFlow-lite 1.13 and custom C libraries for model files although NNStreamer itself supports a lot more neural network frameworks (TensorFlow, Caffe2, PyTorch, custom Python codes, Movidius NCSDK, and so on). However, device developers may install such frameworks and enable ML API sets to utilize such frameworks.
+- **Media Vision**
+Starting with BarCode recognition and generation in Tizen 2.4, detection, recognition and tracking of Face and images have been added to MediaVision. They are based on traditional Computer Vision technology and have been able to guarantee some execution speed on low-end hardware. Recently, the rapid development of AI technology has increased the demand for DNN-based vision technologies, and MediaVision has applied the AI Framework to some of the existing and new functions of MediaVision in response to this trend.
```

Comment:
@nawab-samsung The original sentence has the nuance of advances in technology with the flow of time. ("BarCode recognition and generation" of Tizen 2.4, "detection, recognition and tracking of Face and images" of Tizen 5.5)
To maintain that nuance how about adding "since then"?

In Tizen 2.4 release, bar code recognition and generation features were added to Media Vision. Since then in Tizen 5.5 release, features ~~~ are added to Media Vision.

Please check the suggestion and let us know your thought.

Reply:
In Tizen 2.4 release, bar code recognition and generation features were added to Media Vision. Since  Tizen 5.5 release, features such as detection, recognition, and tracking of face and images are added to Media Vision. These features are based on traditional computer vision technology and guarantee good execution speed on low-end hardware. With the rapid development of AI technology, the demand for DNN-based vision technologies has increased. In response to this growing trend, Media Vision has applied the AI Framework to some of the existing and new functions.